.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+# This file was automatically generated by sbt-github-actions using the
+# githubWorkflowGenerate task. You should add and commit this file to
+# your git repository. It goes without saying that you shouldn't edit
+# this file by hand! Instead, if you wish to make changes, you should
+# change your sbt build configuration to revise the workflow description
+# to meet your needs, then regenerate this file.
+
+name: Continuous Integration
```

Comment:
Why github instead of AWS codebuild?

Reply:
I figured that we might not want to use code build for open-source repos as it's more expensive, moreover adding GitHub actions was just a quick win with one sbt plugin.